code/iaas/model/src/main/java/io/cattle/platform/core/dao/impl/LoadBalancerTargetDaoImpl.java

```diff
@@ -111,4 +111,18 @@ public LoadBalancerTarget getLbIpAddressTarget(long lbId, String ipAddress) {
                         .and(LOAD_BALANCER_TARGET.LOAD_BALANCER_ID.eq(lbId)))
                 .fetchInto(LoadBalancerTargetRecord.class);
     }
+
+    @Override
+    public List<? extends LoadBalancerTarget> getLoadBalancerActiveInstanceTargets(long lbId) {
```

Comment:
Super minor, but the name is very similar to the one above: 'getLoadBalancerActiveTargetInstances'.  I guess since this function name ends with *Targets, it's not too bad.